.gitignore

```diff
@@ -51,3 +51,6 @@ build-iPhoneSimulator/
 
 # rubocop
 .rubocop-https---raw-githubusercontent-com-Invoca-style-guide-master-ruby--rubocop-yml
+
+# appraisal
+gemfiles/*.lock
```

Comment:
Interesting--I'd missed this subtlety. The appraisals generate a fresh `Gemfile.lock` each time?

Reply:
yea, this is a best practice set by the appraisals gem to make sure that the appraisal is always tested against the most recent version of the lock.  it leans very heavily on semantic versioning and locking.